### ISSUE TITLE
Clarify relationship to uglify and uglify-es

### ISSUE THREAD

**Issue by Morgul** [fabiosantoscode/terser#53]
(This isn't exactly a code related question, so I didn't follow the template.)

I was hoping the maintainers could clarify how `terser` relates to `uglify` and `uglify-es`. From what I can tell, `terser` is just `uglify-es` renamed. But, I'm not sure if terser is a temporary fork until the es6+ code gets worked out and will eventually be merged back into `uglify`; or are they two separate projects and `terser` just submits patches upstream occasionally, or has `uglify` decided to never support es6+ code, or...?

Basically, from my perspective as a tool consumer who has a long history with `uglify`, but now sees my tools switching to `uglify-es` and now `terser`, I just would like to know what the history is, and what project I should be following, and why uglify didn't just get es6+ support, since it still seems to be maintained...

It would be really great if some kind of a summary/explaination made it on the project Readme as well. 🙂

**Comment by kzc** [comment]
Hopefully I won't have to rehash this again.

* `terser` is an independent maintained fork of `uglify-es` supporting ES6+ with dozens of bug fixes.  It is not temporary.
* `uglify-js` only supports ES5 and is still actively maintained.
* `uglify-es` supports ES6+ but is no longer maintained.
* the `uglify-js` and `terser` projects are maintained by different people. 
* `terser` should be able to minify any ECMAScript code that `uglify-js` can handle.
* a number of ES5 bug fixes and features have been ported from `uglify-js` to `terser`.
* in theory `uglify-js` could port bug fixes and features from `terser`, although it has only ported a few unit tests at the time of this writing.

> has uglify decided to never support es6+ code, or...?

Don't know. It's up to the `uglify-js` maintainer. It won't affect `terser`.
